docs/developer-guide/ksqldb-clients/java-client.md

```diff
@@ -768,13 +768,21 @@ System.out.println(description.name()
 );
 ```
 
-Use variable substitution<a name="variable-substitution"></a>
+Define variables for substitution<a name="variable-substitution"></a>
 ---------------------------------------------------------------
 
-Starting with ksqlDB 0.18.0, users can define session variables by calling the `define()` method and
-reference them in other fuctions by wrapping the variable name in `${}`. The `undefine()` method
-undefines a session variable, and `getVariables()` returns a map of the currently defined variables
-and their values.
+Starting with ksqlDB 0.18.0, users can define session variables by calling the [`define()`](/api/io/confluent/ksql/api/client/Client.html#define(java.lang.String,boolean,java.lang.Object)) method and
+reference them in other functions by wrapping the variable name in `${}`. The [`undefine()`](/api/io/confluent/ksql/api/client/Client.html#undefine(java.lang.String)) method
+undefines a session variable, and [`getVariables()`](/api/io/confluent/ksql/api/client/Client.html#getVariables()) returns a map of the currently defined variables
+and their values. Substitution is supported for the following functions:
+* `streamQuery`
+* `executeQuery`
+* `terminatePushQuery`
```

Comment:
Does the close query endpoint actually support variable substitution? I didn't think so -- confused how this could work if not.

Reply:
My bad, forgot it was a different endpoint. Removed.